src/rest/CommunityEndpoints.js

```diff
@@ -19,7 +19,7 @@ import {
 } from 'ethers'
 import debugFactory from 'debug'
 
-import * as CommunityProduct from '../../contracts/CommunityProduct.json'
+import * as CommunityProduct from '../../contracts/DataunionVault.json'
```

Comment:
`Dataunion` vs `DataUnion`?

Reply:
Here's what I've been using:
- `DataUnion` for class names
- `dataUnion` for variables
- In URL routes, just `dataunion` to avoid the annoyances of mixed case and punctuation